rules/autocorrect-in-outlook

```diff
@@ -0,0 +1,42 @@
+type: rule
+title: Do you know how to use Autocorrect in Outlook to insert common links you often send?
```

Comment:
Hi @ae-cher-ssw 

This title is too long.

- [ ] 1. What about changing to _"Do you use Autocorrect in Outlook to insert links you often send?"_

Reply:
Done

Reply:
Done